packages/datadog-plugin-jest/test/circus.spec.js

```diff
@@ -148,7 +148,8 @@ describe('Plugin', function () {
                 [TEST_SOURCE_FILE]: 'packages/datadog-plugin-jest/test/jest-test.js',
                 [TEST_TYPE]: 'test',
                 [JEST_TEST_RUNNER]: 'jest-circus',
-                [TEST_CODE_OWNERS]: JSON.stringify(['@DataDog/dd-trace-js']), // reads from dd-trace-js
+                // reads from dd-trace-js' CODEOWNERS
```

Comment:
Can these instead be a fixture file instead of reading the real CODEOWNERS file?

I think it'll get annoying when we start diverging code between branches, might cause conflicts, e.g.
https://github.com/DataDog/dd-trace-js/pull/3474

Reply:
true. Setting up a fixture is a bit annoying, so what I ended up doing is just checking that it's a string and contains `@datadog` (which is pretty safe)